pkg/virt-controller/services/virtiofs.go

```diff
@@ -154,22 +154,14 @@ func virtioFSMountPoint(volume *v1.Volume) string {
 	return volumeMountPoint
 }
 
-func generateContainerFromVolume(config *virtconfig.ClusterConfig, volume *v1.Volume, image string, resources k8sv1.ResourceRequirements) k8sv1.Container {
+func generateContainerFromVolume(volume *v1.Volume, image string, resources k8sv1.ResourceRequirements) k8sv1.Container {
 
 	socketPathArg := fmt.Sprintf("--socket-path=%s", virtiofs.VirtioFSSocketPath(volume.Name))
 	sourceArg := fmt.Sprintf("--shared-dir=%s", virtioFSMountPoint(volume))
-	args := []string{socketPathArg, sourceArg, "--cache=auto"}
 
-	securityProfile := restricted
-	sandbox := "none"
-	if virtiofs.CanRunWithPrivileges(config, volume) {
-		securityProfile = privileged
-		sandbox = "chroot"
-		args = append(args, "--xattr")
-	}
```

Comment:
I cannot comment on non-touched code (very bad UI github...)

since we remove the privileges, we should remove: `virtiofsCredential()`, and `virtiofsCapabilities()`.

and we should remove:
```
var privilegedId = int64(util.RootUser)
var restrictedId = int64(util.NonRootUID)
type securityProfile uint8
const (
	restricted securityProfile = iota
	privileged
)


func isRestricted(profile securityProfile) bool {...}
func isPrivileged(profile securityProfile) bool {...}
```

We need to change  `securityContextVirtioFS()`, to unconditionally return the current "restricted" profile, like:

```
func securityContextVirtioFS(profile securityProfile) *k8sv1.SecurityContext {

	return &k8sv1.SecurityContext{
		RunAsUser:                  int64(util.NonRootUID),
		RunAsGroup:               int64(util.NonRootUID),
		RunAsNonRoot:           pointer.P(true),
		AllowPrivilegeEscalation: pointer.P(false),
		Capabilities:  &k8sv1.Capabilities{
		        Drop: []k8sv1.Capability{
			        "ALL",
		        },
	        },
	}
}
```
or keep `virtiofsCapabilities()` if you think is more clear (dropping ALL of course)

so we can change this:
```

	return k8sv1.Container{
               ...
		SecurityContext: securityContextVirtioFS(securityProfile),
	}
```
to this:
```
	return k8sv1.Container{
               ...
		SecurityContext: securityContextVirtioFS(),
	}
```

Reply:
Done.